src/field_theory/separable_degree.lean

```diff
@@ -0,0 +1,81 @@
+/-
+Copyright (c) 2021 Jakob Scholbach. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jakob Scholbach
+-/
+import field_theory.separable
+import algebra.algebra.basic
+import data.polynomial.degree
+import algebra.char_p.exp_char
+/-!
+# Separable degree
+
+This file contains basics about the separable degree of a polynomial.
+
+## Main results
```

Comment:
Insert a blank line here? Not sure it makes much difference but I think it's what is usually done.

Reply:
Done

Reply:
I am sorry, I must have overwritten my edits. Now (re)done!